Client/Frontend/Login Management/LoginListViewModelHelper.swift

```diff
@@ -5,9 +5,21 @@
 import Storage
 import Shared
 
-// MARK: - Data Source
-class LoginListDataSourceHelper {
```

Comment:
Why was this name changed? I think using `LoginListDataSourceHelper` was a bit more specific, since this is directly tableview and DataSource related, as opposed to being a general purpose helper for the view model.

Reply:
I changed it because as I was working with the functions in it for testing, the scope seemed very limited to formatting things for the view model to process. My line of thought was that something named after the DataSource should be more closely linked to it. `LoginDataSource.swift` itself uses functions in `LoginListViewModel.swift` but not `LoginListDataSourceHelper.swift`.